src/model/document.js

```diff
@@ -307,15 +307,15 @@ export default class Document {
 		if ( this._hasDocumentChangedFromTheLastChangeBlock() ) {
 			this._callPostFixers( writer );
 
+			// Refresh selection attributes according to the final position in the model after the change.
+			this.selection.refreshAttributes();
```

Comment:
Shouldn't it be both before and after `change` event?

Reply:
Theoretically, it should be not necessary. Changing the model on `change` event is not recommended, post fixers are the last who should change the model.

Reply:
Or I missed something?